terraform/examples/io500/config/config.sh

```diff
@@ -67,7 +65,7 @@ export TF_VAR_server_template_name="${DAOS_SERVER_BASE_NAME}"
 export TF_VAR_server_mig_name="${DAOS_SERVER_BASE_NAME}"
 export TF_VAR_server_machine_type="${DAOS_SERVER_MACHINE_TYPE}"
 export TF_VAR_server_os_project="${TF_VAR_project_id}"
-export TF_VAR_server_os_family="daos-server-centos-7"
+export TF_VAR_server_os_family="daos-server-io500-centos-7"
```

Comment:
@markaolson   Why do we need a daos-server-io500-centos-7 image? 

IO500 is a client application  a special server should not be needed.

Reply:
When there are multiple people working in the same project one person might be testing IO500 on DAOS 2.0.0 and another person may be testing the terraform/examples/full_cluster_setup or testing images with new versions in some other way. This actually happened with Johann and I a couple of times where my images messed up what he was doing. This is the first step in fixing the issue.  When you run the IO500 the server and client images need to match. If you run the --version and pass a new version with the --force option you do not want to blow away your main daos-server-centos-7 image.  So this just ensures that the images you are using are specifically built for the IO500 example and don't interfere with anything else.  We still have a larger issue we need to address in a future PR.  We need a way to tag/label the images with the version number so that the scripts can check for existing images with a specific version.  Topic for another day.

Reply:
We have labels for the MIG groups on the deploy side and this works well. 

Adding a custom label for Machines Images seems a good path.  Hard coding "IO500" seem narrow functionally.

Reply:
@markaolson had a good chat today.  I better understand what is going on.